virttest/utils_conn.py

```diff
@@ -822,11 +825,10 @@ def conn_setup(self):
             # From libvirt 5.6, libvirtd is using systemd socket activation
             # by default
             if libvirt_version.version_compare(5, 6, 0, session):
-                session.cmd("systemctl stop libvirtd.socket")
+                # Before start libvirtd-tcp.socket, user must stop libvirtd.
+                # After libvirtd-tcp.socket is tarted, user mustn't start libvirtd.
```

Comment:
s/`is tarted,`/`is started,`

Reply:
@sathnaga , thanks for your pointing out. I updated it.